### ISSUE TITLE
default html templates for a specific langage

### ISSUE THREAD

**Issue by farvardin** [YunoHost-Apps/mailman_ynh#16]
bonjour,

I've installed mailman, it looks cool, however I was confused because when I create a new list in a specific language (French), all the subscription templates are in English, which is not suitable for our French users.

I was beginning to translate the first sample, when I thought it might already translated. I've found them in /usr/share/doc/mailman/examples/fr/ and in /usr/share/mailman/fr/ (they look the same).

So to get everything translated in my list, I've done this from the command line:

sudo cp  /usr/share/doc/mailman/examples/fr/* /var/lib/mailman/lists/ma_liste/fr/

is it a known limitation of mailman?

**Comment by alexAubin** [comment]
Uuuuuh I don't know, naively I would thing there's a setting in the mailman interface for this list to be switched to french by default ..?

**Comment by farvardin** [comment]
yes, the interface is in French, and I've selected French everywhere. Yet, the default template used is in English.

It was rather easy to switch everything to French using the command line, but it was not really "user friendly" (I was thinking about some other YNH users)

I'll probably report it upstream to Mailman
